addons/l10n_eu_service/models/res_company.py

```diff
@@ -13,7 +14,7 @@ def _map_all_eu_companies_taxes(self):
         ''' Identifies EU companies and calls the _map_eu_taxes function
         '''
         eu_countries = self.env.ref('base.europe').country_ids
-        companies = self.search([('country_id','in', eu_countries.ids)])
+        companies = self.search([]).filtered(lambda c : c.partner_id.country_id.id in eu_countries.ids)
```

Comment:
This is the a side fix to correct the l10n builds.
`country_id` is a computed field with `store=false`